less/theme.less

```diff
@@ -836,7 +829,13 @@ footer {
 
     a {
         letter-spacing: 1.1px;
-        margin-bottom: -4px;
```

Comment:
`.page-tabs` styles were tweaked in an earlier tab to not use negative margins, so these rules above are not necessary, and in fact were causing some clipping issues on Run Details.